components/locale.js

```diff
@@ -61,14 +61,26 @@ function getInitialLocaleProps({req} = {}) {
   return Promise.resolve({locale})
 }
 
-// eslint-disable-next-line complexity
 function getContent(locale, options) {
+  const fallbackContent = content('', options)
+  if (locale === fallbackLocale) {
+    return fallbackContent
+  }
+  const localeContent = content(`${locale}/`, options)
+  return Object.keys(fallbackContent).reduce((cont, key) => {
```

Comment:
This might be tricky tho.

For example if the default translation is
```
module.exports = {
  sections: [
    require('./foo'), 
    require('./bar'), 
  ]
}
```
and in a translate file it is defined as 
```
module.exports = {
  sections: [
    require('./bar'), 
    require('./foo'), 
  ].filter(mod => Boolean(mod.title)),
}
```
Is there a way to link up `foo` with `foo` and `bar` with `bar`?